okhttp-testing-support/src/main/java/okhttp3/OkHttpClientTestRule.kt

```diff
@@ -75,21 +88,29 @@ class OkHttpClientTestRule : TestRule {
   private fun ensureAllTaskQueuesIdle() {
     for (queue in TaskRunner.INSTANCE.activeQueues()) {
       assertThat(queue.awaitIdle(TimeUnit.MILLISECONDS.toNanos(1000L)))
-          .withFailMessage("Queue still active after 1000ms")
+          .withFailMessage("Queue still active after 1000 ms")
           .isTrue()
     }
   }
 
   override fun apply(base: Statement, description: Description): Statement {
     return object : Statement() {
       override fun evaluate() {
+        val defaultUncaughtExceptionHandler = Thread.getDefaultUncaughtExceptionHandler()
```

Comment:
Do any of our build environments run tests concurrently in the same process? Gradle, Intellij?

Reply:
Nope! And if ever we want to, multiple processes is the right way. We have lots of static state in our tests, including system properties and security providers.